dfsclient/src/main/java/org/hpccsystems/dfs/client/HpccRemoteFileReader.java

```diff
@@ -246,6 +257,51 @@ public HpccRemoteFileReader(DataPartition dp, FieldDef originalRD, IRecordBuilde
         openTimeMs = System.currentTimeMillis();
     }
 
+    private boolean retryRead()
+    {
+        if (retryCount < 3)
```

Comment:
shouldn't the max retry count be configurable?

Reply:
@rpastrana Implemented code review changes, will squash if it looks good